src/main/java/com/authservice/service/JwtService.java

```diff
@@ -0,0 +1,73 @@
+package com.authservice.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtService {
+
+    public static final String SECRET = "5367566B59703373367639792F423F4528482B4D6251655468576D5A71347437";
+
```

Comment:
We should write encryption class , that encrypt the key and this should be used as a signing key .. Remove hard code.

Reply:
updated code chnages : Completed  and added JwtEncryption class and add method encodeSecretKey()